src-built-in/components/advancedAppCatalog/src/app.jsx

```diff
@@ -173,6 +168,47 @@ export default class AppMarket extends React.Component {
 
 		return page;
 	}
+
+	/**
+	 * Returns the activeApp from the Store
+	 *
+	 * @returns {string} activeApp
+	 * @memberof AppMarket
+	 */
+	getActiveApp() {
+		return storeActions.getActiveApp();
+	}
+
+	/**
+	 * Checks if we have an activeApp defined
+	 *
+	 * @returns {boolean}
+	 * @memberof AppMarket
+	 */
+	isActiveApp() {
+		return storeActions.getActiveApp() !== null;
```

Comment:
```suggestion
		return storeActions.getActiveApp();
```
I don't think you need `!== null` here. The `getActiveApp()` method will either return an active app or nothing (which can be evaluated as Boolean)

Reply:
I replicated the logic that was in the component. Really, if what you're saying is the case, then this entire function is superfluous as we could just call getActiveApp().

Stylistically, when the difference is this small, I prefer to be more declarative in what the code is doing instead of relying on assuming the system will do something. The intent is clearer here for the next programmer.